include/mbedtls/ssl.h

```diff
@@ -359,12 +359,19 @@
 #if !defined(MBEDTLS_PSK_MAX_LEN)
 #define MBEDTLS_PSK_MAX_LEN            32 /* 256 bits */
 #endif
+#define MBEDTLS_PSK_LEN_BYTES       2
+
+#define MBEDTLS_RSA_PMS_LEN_BYTES   2
+#define MBEDTLS_RSA_PMS_VERSION_LEN 2
+#define MBEDTLS_RSA_PMS_RANDOM_LEN  46
+#define MBEDTLS_RSA_PMS_LEN                                             \
+    ( MBEDTLS_RSA_PMS_VERSION_LEN + MBEDTLS_RSA_PMS_RANDOM_LEN )
 
 /* Dummy type used only for its size */
 union mbedtls_ssl_premaster_secret
 {
 #if defined(MBEDTLS_KEY_EXCHANGE_RSA_ENABLED)
-    unsigned char _pms_rsa[48];                         /* RFC 5246 8.1.1 */
+    unsigned char _pms_rsa[ MBEDTLS_RSA_PMS_LEN ];     /* RFC 5246 8.1.1 */
```

Comment:
Inconsistent indentation throughout the PR - with and without spaces inside the [].

Reply:
Done.